test/Window/WindowBase.test.cpp

```diff
@@ -104,14 +105,38 @@ TEST_CASE("[Window] sf::WindowBase", runDisplayTests())
 
     SECTION("pollEvent()")
     {
-        sf::WindowBase windowBase;
-        CHECK(!windowBase.pollEvent());
+        SECTION("Uninitialized window")
+        {
+            sf::WindowBase windowBase;
+            CHECK(!windowBase.pollEvent());
+        }
     }
 
     SECTION("waitEvent()")
     {
-        sf::WindowBase windowBase;
-        CHECK(!windowBase.waitEvent());
+        SECTION("Uninitialized window")
+        {
+            sf::WindowBase windowBase;
+            CHECK(!windowBase.waitEvent());
+        }
+
+        SECTION("Initialized window")
+        {
+            sf::WindowBase windowBase(sf::VideoMode({360, 240}), "WindowBase Tests");
+
+            constexpr auto timeout = sf::milliseconds(250);
```

Comment:
Can we take a lower value? I wouldn't want to delay the test run by a quarter of a second. This also increases the risk that on a real system you end up getting an event within those 250ms.

Reply:
I will tweak the values.